server/src/test/java/com/example/tyfserver/member/MemberAcceptanceTest.java

```diff
@@ -334,4 +338,35 @@ public void registerAccount() {
 
         assertThat(response.statusCode()).isEqualTo(200);
     }
+
+    @Test
+    @DisplayName("정산 요청 - 정산 금액이 만원 이하인 경우")
+    public void requestExchangeAmountLessThanLimit() {
+        //given
+        SignUpResponse signUpResponse = 회원가입_후_로그인되어_있음("email@email.com", "KAKAO", "nickname", "pagename");
+        PaymentPendingResponse pendingResponse = 페이먼트_생성(10000L, "donator@gmail.com", "pagename").as(PaymentPendingResponse.class);
+        후원_생성("impUid", pendingResponse.getMerchantUid().toString()).as(DonationResponse.class).getDonationId();
+
+        //when
+        ErrorResponse errorResponse = 정산_요청(signUpResponse.getToken()).as(ErrorResponse.class);
+
+        //then
+        assertThat(errorResponse.getErrorCode()).isEqualTo(ExchangeAmountException.ERROR_CODE);
+    }
+
+    @Test
+    @DisplayName("정산 요청 - 이미 정산 요청을 한 상태인 경우")
+    public void requestExchangeAlreadyRequest() {
+        //given
+        SignUpResponse signUpResponse = 회원가입_후_로그인되어_있음("email@email.com", "KAKAO", "nickname", "pagename");
+        PaymentPendingResponse pendingResponse = 페이먼트_생성(10000L, "donator@gmail.com", "pagename").as(PaymentPendingResponse.class);
+        후원_생성("impUid", pendingResponse.getMerchantUid().toString()).as(DonationResponse.class).getDonationId();
+
+        //when
+        정산_요청(signUpResponse.getToken()).as(ErrorResponse.class);
```

Comment:
난 이 테스트가 실패하는데, 혹시 다른사람들은 괜찮아?? 
에러코드가 `exchange-002`가 발생하고 있네 🤔

에러 메시지랑 코드 확인해보니까  `정산 요청 금액이 10,000을 초과`하도록 만들면 테스트의 본래 의도대로 진행이 될 것 같아!
(그런데 AcceptanceTest에서 
```java
when(paymentServiceConnector.requestPaymentInfo(any(UUID.class)))
                .thenAnswer(invocation -> new PaymentInfo(invocation.getArgument(0), PaymentStatus.PAID, 10000L,
                        "pagename", "impUid", "module"));
```
같이 mocking을 하고 있어서, 값 변경이 쉽지가 않네 🤔
일단 아래와 같이 코드를 변경하고, 모킹을 변경해야할거 같은데 🥲

```suggestion
        PaymentPendingResponse pendingResponse = 페이먼트_생성(20000L, "donator@gmail.com", "pagename").as(PaymentPendingResponse.class);
        후원_생성("impUid", pendingResponse.getMerchantUid().toString()).as(DonationResponse.class).getDonationId();

        //when
        정산_요청(signUpResponse.getToken());
```

Reply:
잉 깨재ㅕ? 나 할때에는 안깨졌었는데 뭐 다른거 수정하면서 깨졋나보다. 저거는 일부러 1000을 10000으로 수정해둔거야 만원이하일떄에 거부되는 케이스 테스트도 해봐야해서 1번 후원하면 해당 익셉션이 발생하고 2번 후원(총 2만원) 하게되면 통과하게끔 하려구